contracts/test/_metastrategies-fixtures.js

```diff
@@ -0,0 +1,221 @@
+const { loadFixture } = require("ethereum-waffle");
+const { ousdUnits } = require("./helpers");
+const { convexMetaVaultFixture } = require("./_fixture");
+
+// NOTE: This can cause a change in setup from mainnet.
+// However, mint/redeem tests, without any changes, are tested
+// in vault.fork-test.js, so this should be fine.
+
+async function withDefaultStrategiesSet() {
+  const fixture = await loadFixture(convexMetaVaultFixture);
+
+  const { vault, governor, usdt, usdc, OUSDmetaStrategy } = fixture;
+
+  await vault
+    .connect(governor)
+    .setAssetDefaultStrategy(usdt.address, OUSDmetaStrategy.address);
+
+  await vault
+    .connect(governor)
+    .setAssetDefaultStrategy(usdc.address, OUSDmetaStrategy.address);
+
+  return fixture;
+}
+
+async function withBalancedMetaPool() {
+  const fixture = await loadFixture(withDefaultStrategiesSet);
+
+  await balanceMetaPool(fixture);
+
+  return fixture;
+}
+
+async function balanceMetaPool(fixture) {
```

Comment:
Good point, since everything involved is a stablecoin, I just assumed the difference would be negligible (if not equal). But totally agree that we should be using dollar values instead, will look into it

Reply:
Cool, [the virtual price](https://curve.readthedocs.io/factory-pools.html#StableSwap.get_virtual_price) on each metapool is a 1e18 denominated value. You can use StableMath's mulTruncate to get dollar value: 
```
dollarValue = lpAmount.mulTruncate(virtualPrice)
```

Reply:
This is done